### PR TITLE
Feature_Card_Globale_Review

### PR DESCRIPTION
**Pull request by JulieCombeau**
Creating components CardReview and CardReviewCaroulsel to display movie comments defined

### CODE REVIEW

**Comment by JulieCombeau** [comment]
[[Feature_Card_Globale_Review]](https://trello.com/c/Ziel0kF6/24-featurecardglobalereview)

File: backend/.eslintrc.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "sourceType": "module"
   },
   "rules": {
+    "linebreak-style": ["error", "unix","windows"],
```
Comment (lchastanet) [inline backend/.eslintrc.json:12]: This line shouldn't be there anymore.

**Review by lchastanet — CHANGES_REQUESTED** [review]
Just remove the line in the Eslint config file and its ok 👌🏻

**Review by lchastanet — APPROVED** [review]
Nickel

**Pull request merged**